### PR TITLE
Change dataloader func and add barcharts for results

### PR DESCRIPTION
**Pull request by rnyak**
This PR updates `03-Session-based-recsys.ipynb` nb with the following changes:

-  load NVT dataloader from schema
-  visualize accuracy results with bar charts

### CODE REVIEW

File: examples/tutorial/03-Session-based-recsys.ipynb [hunk 1]
```diff
@@ -393,80 +392,45 @@
...
     "\n",
-    "    loader = NVTDataLoader(\n",
-    "        dataset=dataset,\n",
-    "        batch_size= bs,\n",
-    "        shuffle=False,\n",
-    "        cats=x_cat_names,\n",
-    "        conts=x_cont_names,\n",
-    "        device=0,\n",
-    "        labels=[],\n",
+    "    return NVTabularDataLoader.from_schema(\n",
```
Comment (benfred) [inline examples/tutorial/03-Session-based-recsys.ipynb:413]: Nice! this looks a lot cleaner

**Review by benfred — APPROVED** [review]
